src/unicode/map.rs

```diff
@@ -0,0 +1,1330 @@
+        0x10C8E => ('\u{10CCE}',).into(),
+        0x10C8F => ('\u{10CCF}',).into(),
+        0x10C90 => ('\u{10CD0}',).into(),
+        0x10C91 => ('\u{10CD1}',).into(),
+        0x10C92 => ('\u{10CD2}',).into(),
+        0x10C93 => ('\u{10CD3}',).into(),
+        0x10C94 => ('\u{10CD4}',).into(),
+        0x10C95 => ('\u{10CD5}',).into(),
+        0x10C96 => ('\u{10CD6}',).into(),
+        0x10C97 => ('\u{10CD7}',).into(),
+        0x10C98 => ('\u{10CD8}',).into(),
+        0x10C99 => ('\u{10CD9}',).into(),
+        0x10C9A => ('\u{10CDA}',).into(),
+        0x10C9B => ('\u{10CDB}',).into(),
+        0x10C9C => ('\u{10CDC}',).into(),
+        0x10C9D => ('\u{10CDD}',).into(),
+        0x10C9E => ('\u{10CDE}',).into(),
+        0x10C9F => ('\u{10CDF}',).into(),
+        0x10CA0 => ('\u{10CE0}',).into(),
+        0x10CA1 => ('\u{10CE1}',).into(),
+        0x10CA2 => ('\u{10CE2}',).into(),
+        0x10CA3 => ('\u{10CE3}',).into(),
+        0x10CA4 => ('\u{10CE4}',).into(),
+        0x10CA5 => ('\u{10CE5}',).into(),
+        0x10CA6 => ('\u{10CE6}',).into(),
+        0x10CA7 => ('\u{10CE7}',).into(),
+        0x10CA8 => ('\u{10CE8}',).into(),
+        0x10CA9 => ('\u{10CE9}',).into(),
+        0x10CAA => ('\u{10CEA}',).into(),
+        0x10CAB => ('\u{10CEB}',).into(),
+        0x10CAC => ('\u{10CEC}',).into(),
+        0x10CAD => ('\u{10CED}',).into(),
+        0x10CAE => ('\u{10CEE}',).into(),
+        0x10CAF => ('\u{10CEF}',).into(),
+        0x10CB0 => ('\u{10CF0}',).into(),
+        0x10CB1 => ('\u{10CF1}',).into(),
+        0x10CB2 => ('\u{10CF2}',).into(),
+        0x118A0 => ('\u{118C0}',).into(),
+        0x118A1 => ('\u{118C1}',).into(),
+        0x118A2 => ('\u{118C2}',).into(),
+        0x118A3 => ('\u{118C3}',).into(),
+        0x118A4 => ('\u{118C4}',).into(),
+        0x118A5 => ('\u{118C5}',).into(),
+        0x118A6 => ('\u{118C6}',).into(),
+        0x118A7 => ('\u{118C7}',).into(),
+        0x118A8 => ('\u{118C8}',).into(),
+        0x118A9 => ('\u{118C9}',).into(),
+        0x118AA => ('\u{118CA}',).into(),
+        0x118AB => ('\u{118CB}',).into(),
+        0x118AC => ('\u{118CC}',).into(),
+        0x118AD => ('\u{118CD}',).into(),
+        0x118AE => ('\u{118CE}',).into(),
+        0x118AF => ('\u{118CF}',).into(),
+        0x118B0 => ('\u{118D0}',).into(),
+        0x118B1 => ('\u{118D1}',).into(),
+        0x118B2 => ('\u{118D2}',).into(),
+        0x118B3 => ('\u{118D3}',).into(),
+        0x118B4 => ('\u{118D4}',).into(),
+        0x118B5 => ('\u{118D5}',).into(),
+        0x118B6 => ('\u{118D6}',).into(),
+        0x118B7 => ('\u{118D7}',).into(),
+        0x118B8 => ('\u{118D8}',).into(),
+        0x118B9 => ('\u{118D9}',).into(),
+        0x118BA => ('\u{118DA}',).into(),
+        0x118BB => ('\u{118DB}',).into(),
+        0x118BC => ('\u{118DC}',).into(),
+        0x118BD => ('\u{118DD}',).into(),
+        0x118BE => ('\u{118DE}',).into(),
+        0x118BF => ('\u{118DF}',).into(),
+        _ => (orig,).into()
```

Comment:
I had that, but realized that needed me to add `#[derive(Clone, Copy)]` to `Fold`, and that's often discouraged for iterators, since you can accidentally `.next()` the wrong copy.